CHANGELOG.md

```diff
@@ -70,6 +70,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Anthropic native Tool calling.
 - Empty `ActionsSubtask.thought` being logged.
+- `RuleMixin` no longer prevents setting `rulesets` _and_ `rules` at the same time.
+- `PromptTask` will merge in its Structure's Rulesets and Rules.
+- `PromptTask` not checking whether Structure was set before building Prompt Stack.
```

Comment:
probably need a breaking change for `all_rulesets -> rulesets`

Reply:
I had left it out since it felt more like an internal change, but I can add it back.

Reply:
might be worth keeping just because it was a public property.